ajax/libs/buy-button-js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "buy-button-js",
+  "filename": "buybutton.js",
+  "author": "Shopify Inc.",
+  "version": "0.4.13",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com:Shopify/buy-button-js.git"
```

Comment:
Please modify this to `https://github.com/Shopify/buy-button-js.git` to enable users to browse it.